src/js/embed.js

```diff
@@ -59,51 +60,106 @@ function getQueryParams() {
     return params;
 }
 
+
 window.init = function init(parentEl) {
     const params = getQueryParams();
-    const { sheet, id, format = 'flat' } = params;
-
-    if (!formats[format]) {
-        throw new Error(`format ${format} is not valid`);
-    }
-    const { template, postRender, preprocess, url } = formats[format];
+    const isTailored = Boolean(params.tailored);
 
     iframeMessenger.enableAutoResize();
     setupVisibilityMonitoring();
-    reqwest({
-        url,
-        type: 'json',
-        crossOrigin: false,
-        success: (res) => {
-            const templateFn = dot.template(template, null, { feedback });
-            const rows = res && res.sheets && res.sheets[sheet];
-            const trackingCode = `brexit__${sheet}__${id}`;
-            let row;
-
-            if (!rows || !rows.length) {
-                throw new Error(`bad JSON response: ${JSON.stringify(res)}`);
-            }
-            rows.forEach((r) => {
-                if (r.id === id) {
-                    row = r;
-                }
-            });
-            if (!row) {
-                throw new Error(`row with id ${id} not found`);
+
+    function getRowById(rows, rowId) {
+        const row = rows.reduce((prev, currentRow) => {
```

Comment:
Yes, `reduce` was introduced in ES5. `find` was finally introduced in ES6 but is not available in all our supported browsers, unless we shim it. Sigh.